README.md

```diff
@@ -14,30 +14,120 @@
   </a>
 </a>
 </p>
+
+<h2 align="center"><a href="https://killercoda.com/open-feature/scenario/flagd-demo" target="_blank">Try flagd in the Killercoda Playground (in browser)</a></h4>
 <!-- markdownlint-enable MD033 -->
 
 ## Features
 
-- OpenFeature compliant with providers available in many languages
-- Multiple flag configuration sources including `files`, `http`, and `Kubernetes`
-- Accessible over gRPC and HTTP
-- Supports subscriptions to real-time flag change events
-- Flexible targeting rules based on [JSON Logic](https://jsonlogic.com/)
-- Lightweight daemon, with an emphasis on performance
-- Native support for metrics using Prometheus
+Flagd is a simple command line tool for fetching and evaluating feature flags.
+
+Think of it as a ready-made, open source, OpenFeature compliant feature flag backend system.
+
+flagd:
+
+- 🌐 OpenFeature compliant and [speaks your language](docs/usage/flagd_providers.md).
+- 🆕 Easy to [extend to new languages](docs/other_resources/creating_providers.md).
+- 🔄 Centralized flag management. Use multiple flag sources and vendors. [Flag values are automatically merged](docs/configuration/flag_configuration_merging.md).
+- 🕒 Feature Flag updates occur in near-realtime (HTTP) or realtime (gRPC)
+- 💪 Contains a [powerful and flexible rule targeting engine](docs/configuration/reusable_targeting_rules.md) and [deterministic percentage-based rollouts](https://github.com/open-feature/flagd/blob/main/docs/configuration/fractional_evaluation.md).
+- 🔦 [Flag statistics and metrics are exposed](https://example.com) and compatible with Prometheus.
```

Comment:
Update the example link.

Reply:
I'll remove it for now as we don't have the documentation ready. I'll create another PR when that doc is ready.